daffodil-test-ibm1/src/test/scala/org/apache/daffodil/TresysTests.scala

```diff
@@ -175,6 +175,7 @@ class TresysTests {
 
   // @Test def test_AP000() { runnerAP.runOneTest("AP000") } // lengthKind endOfParent - DAFFODIL-567
 
+  // Were these tests overlooked while fixing DAFFODIL-341?
```

Comment:
Yes I believe you are correct. If we have coverage elsewhere of catching multiple-assignment and I know we have tests of newVariableInstance elsewhere. Mostlikely we should just drop these tests.

Reply:
Agreed.